core/templates/domain/platform_feature/platform-parameter-filter.model.ts

```diff
@@ -13,14 +13,11 @@
 // limitations under the License.
 
 /**
- * @fileoverview Factory for creating PlatformParameterFilter domain objects.
+ * @fileoverview Frontend Model for Platform Parameter Filter.
```

Comment:
``Frontend model for platform parameter filters.``

The previous wording didn't quite read correctly. Ditto elsewhere in this PR (where applicable).